src/components/elements/dropdown.js

```diff
@@ -99,7 +99,7 @@ const Arrow = styled(Chevron)`
     right: 8px;
     top: 25%;
     transition: transform 0.2s linear;
-    ${props => (props.expanded ? 'transform: rotate(-180deg);' : '')}
+    ${props => (props.expanded ? '' : 'transform: rotate(-180deg);')}
```

Comment:
```suggestion
    ${props => (props.expanded && 'transform: rotate(-180deg);')}
```

Reply:
we should pass a string and compare it with the string value, based on styled component.